File: backend/Models/Note.js
const mongoose = require('mongoose');

const NoteSchema = new mongoose.Schema({
    content: { type: String, required: true }, 
    bookId: { type: String, required: true },  
    dateAdded: { type: Date, default: Date.now }
});

module.exports = mongoose.model('Note', NoteSchema);

File: Models/Note.cs
using System.Text.Json.Serialization;

namespace project.Models;

public class Note
{
    [JsonPropertyName("_id")]
    public string Id { get; set; }

    [JsonPropertyName("bookId")]
    public string BookId { get; set; }

  
    [JsonPropertyName("content")]
    public string Content { get; set; }

    [JsonPropertyName("createdAt")]
    public DateTime DateAdded { get; set; } = DateTime.Now;
}